eng/references.targets

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <ContractDependencyPaths>$(RefPath)</ContractDependencyPaths>
+    <ContractDependencyPaths>$(RefPath);@(ReferencePath)</ContractDependencyPaths>
```

Comment:
I think you need directories, right?

Reply:
sorry don't follow. Are you talking about the item not being available in the property? The item is evaluated inside a target so it is available.

Reply:
`@(ReferencePath)` is the full path to the referenced DLLs, whereas `$(RefPath)` is a directory.  I believe we should be passing the directories, eg `@(ReferencePath->'%(RootDir)%(Directory)')`

Reply:
Got it, will do.

Reply:
(you may need to deduplicate, not sure)